exercises/affine-cipher/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "exercism-javascript",
+  "description": "Exercism exercises in Javascript.",
+  "author": "Tom Pradat",
```

Comment:
Am I supposed to put my name here ?

Reply:
I suppose not, the CI fails if author is not "Katrina Owen"

Reply:
We will use the v3 format to get your authorship in!